scripts/genmsgpack.lua

```diff
@@ -43,7 +43,16 @@ c_proto = Ct(
 grammar = Ct((c_proto + c_comment + c_preproc + ws) ^ 1)
 
 -- we need at least 4 arguments since the last two are output files
-assert(#arg >= 3)
+if arg[1] == '--help' then
```

Comment:
I think the old name of this file was better, it does not handle msgpack-rpc dispatch specifically, but all api dispatch including dispatch for viml. (Though some name with "api" in it like genapidispatch or genapiwrappers might be even better)